offense.md

```diff
@@ -199,4 +199,4 @@ All salary figures are estimates based on market data and may vary significantly
 
 This is an open-source effort to standardize security titles. [Join the discussion](https://github.com/mubix/securitytitles.com/discussions) to help improve these frameworks.
 
-</div>
\ No newline at end of file
+</div>
```

Comment:
Genuinely no idea what this line diff is for, trailing newline maybe? I just made this patch in the github web editor.